todo_app/app.py

```diff
@@ -3,31 +3,28 @@
 from flask import request
 
 from todo_app.flask_config import Config
-from todo_app.data.session_items import *
+from todo_app.trello_api import get_todo_cards, create_todo_card, move_card_to_done_list
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
-
 @app.route('/', methods=['GET'])
 def index():
-    sorted_items = sorted(get_items(), key=lambda item: item['status'], reverse=True)
+    todo_cards = get_todo_cards()
+    items = [{'id': card['id'], 'status': 'Not Started', 'title': card['name']} for card in todo_cards]
+    sorted_items = sorted(items, key=lambda item: item['status'], reverse=True)
     return render_template("index.html", items=sorted_items)
 
 @app.route('/', methods=['POST'])
 def new_item():
     item_title = request.form['item_title']
-    add_item(item_title)
+    create_todo_card(item_title)
     return redirect(url_for('index'))
 
 
-@app.route('/complete', methods=['POST'])
-def complete_item():
-    completed = request.form.getlist('completed')
-    for c in completed:
-        item = get_item(c)
-        item['status'] = 'Completed'
-        save_item(item)
+@app.route('/complete/<item_id>', methods=['GET'])
```

Comment:
Why have you changed this to a GET route? It changes the state of your app so it is better to use POST

Reply:
Sorry I resolved this on the 27th of May but didn't click the Resolve conversation button.